Description: Upstream changes introduced in version 1.4.0-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 google-mock (1.4.0-3) unstable; urgency=low
 .
   * Quick fix for test failure on big endian architectures (Closes: #580514)
 .
 The person named in the Author field signed this changelog entry.
Author: Fredrik Hallenberg <hallon@debian.org>
Bug-Debian: http://bugs.debian.org/580514

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- google-mock-1.4.0.orig/test/gmock-printers_test.cc
+++ google-mock-1.4.0/test/gmock-printers_test.cc
@@ -36,6 +36,7 @@
 #include <gmock/gmock-printers.h>
 
 #include <ctype.h>
+#include <endian.h>
 #include <limits.h>
 #include <string.h>
 #include <algorithm>
@@ -86,7 +87,8 @@ namespace foo {
 // A user-defined unprintable type in a user namespace.
 class UnprintableInFoo {
  public:
-  UnprintableInFoo() : x_(0x12EF), y_(0xAB34), z_(0) {}
+   
+  UnprintableInFoo() : x_(htole32(0x12EF)), y_(htole32(0xAB34)), z_(0) {}
  private:
   testing::internal::Int32 x_;
   testing::internal::Int32 y_;
